docs/book/how-to/infrastructure-deployment/infrastructure-as-code/README.md

```diff
@@ -94,7 +94,7 @@ You can learn more about how to generate a `ZENML_API_KEY` via service accounts
 [here](https://docs.zenml.io//how-to/manage-zenml-server/connecting-to-zenml/connect-with-a-service-account).
 
 
-If you're using a ZenML Pro server, you will need to create an organization-level service account and an API key for it. You can find more about organization-level service accounts and API keys [here](https://docs.zenml.io/pro/core-concepts/service-accounts).
+If you're using a ZenML Pro server, you will need to create a Personal Access Token or an organization-level service account and an API key for it. You can find more about Personal Access Tokens [here](https://docs.zenml.io/pro/core-concepts/personal-access-tokens) and organization-level service accounts and API keys [here](https://docs.zenml.io/pro/core-concepts/service-accounts).
```

Comment:
URL is not right IMO https://zenml-io.gitbook.io/stefan/zenml-pro/access-management/personal-access-tokens